testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MainActivity.java

```diff
@@ -226,6 +228,8 @@ IAccount getAccount(final String loginHint) {
         for (final IAccount account : mApplication.getAccounts()) {
             if (account.getUsername().equals(loginHint.trim().toLowerCase())) {
                 return account;
+            } else if (account.getAccountIdentifier().getIdentifier().equals(loginHint.trim().toLowerCase())) {
```

Comment:
I don't think this _should_ work: this value is either going to be a GUID or an pair-wise ID: I don't know under what circumstances this could reliably match a login_hint but this seems flawed 🤔

Reply:
The oid is returned as string here account.getAccountIdentifier().getIdentifier().

Reply:
In IAccount.java

Reply:
The oid is returned as a string in the code `account.getAccountIdentifier().getIdentifier()`

Reply:
I will remove this part for now. 'cause I found some other bug for testapp. Let's just leave it as it is for now.